src/ima-state-machine.js

```diff
@@ -314,6 +314,11 @@ function onAdBreakEnd(options: Object, adEvent: any): void {
     }
     this._hideAdsContainer();
     this._maybeSetVideoCurrentTime();
+    const shouldPlayerSeekToLiveEdge =
+      this.player.isLive() && !this.config.disableMediaPreload && getAdBreakType.call(this, adEvent) === AdBreakType.PRE;
+    if (shouldPlayerSeekToLiveEdge) {
+      this.player.seekToLiveEdge();
```

Comment:
this is problematic in case there is a bumper after the pre roll (before the live loaded)